automation_tools/satellite6/upgrade/capsule.py

```diff
@@ -97,9 +90,8 @@ def satellite6_capsule_upgrade(cap_host):
     # org, and content host in one), the content host will be unregistered as
     # part of the upgrade process.
     if to_version == '6.2':
-        ak_name = os.environ.get('CAPSULE_SUBSCRIPTION').split(',')[2].strip(
-            ) if os.environ.get('CAPSULE_SUBSCRIPTION') else os.environ.get(
-            'RHEV_CAPSULE_AK')
+        ak_name = os.environ.get('CAPSULE_AK') if os.environ.get(
+            'CAPSULE_AK') else os.environ.get('RHEV_CAPSULE_AK')
```

Comment:
If you start to check a spread of conditions like this, check Chain Map once it can get rid of conditionals on your code: https://docs.python.org/3/library/collections.html#collections.ChainMap. I checked the same on other parts of code, but not so many conditions that worth it. So I am mentioning just in case you don't know about it.